examples/meta/src/converter/multidimensional_scaling.sg

```diff
@@ -0,0 +1,32 @@
+CSVFile data_fname("../../data/fm_train_real.dat")
+
+#![create_features]
+RealFeatures features(data_fname)
+#![create_features]
+
+#![calculate_distances_before]
+EuclideanDistance distance_before()
+distance_before.init(features, features)
+#![calculate_distances_before]
+
+#![apply_mds]
+MultidimensionalScaling converter()
+converter.set_target_dim(2)
+converter.set_landmark(False)
+RealFeatures embedding = converter.apply(features)
+#![apply_mds]
+
+#![calculate_distances_after]
+EuclideanDistance distance_after()
+distance_after.init(embedding, embedding)
+#![calculate_distances_after]
+
+#![calculate_distance_matrices]
+RealMatrix distance_matrix_after = distance_after.get_distance_matrix()
+RealMatrix distance_matrix_before = distance_before.get_distance_matrix()
+#![calculate_distance_matrices]
+
+# additional integration testing variables
+#bool output = ((numpy.linalg.norm(distance_matrix_after-distance_matrix_before) / numpy.linalg.norm(distance_matrix_before)) < 1e-6)
```

Comment:
I believe that the purpose of this boolean variable is to test whether the distances are preserved after the transformation, which is already ensured by the `distance_preserving` test in "MultidimensionalSaling_unittest.cc". So, no need for another one.